### DISCUSSION TITLE
TB 115.4.2

### DISCUSSION THREAD

**Discussion by TJR032354** [kewisch/gdata-provider#680] [Support] [answered]
Just wondering if this add-on is still necessary after upgrading to TB 115.x  Didn't TB 115 include full Google calendar integration ??

**Comment by jannek-aalto** [comment]
Well, I certainly can't see Google calendars shared with me with a vanilla TB 115.4.2.

**Comment by kewisch** [comment, marked as answer]
Thunderbird 115 certainly has some built-in support using CalDAV. This add-on uses a different method to access the calendar, and provide some UI improvements on top. Please use whatever works better for you :-)
